README.md

```diff
@@ -58,52 +39,30 @@ Unlike other SR v4 boards, the STM32 on the motor board does not directly commun
 
 The FTDI Chip has vendor ID `0403` and product ID `6001`. It can be further filtered by the USB `product` field.
 
-Given that the appropriate drivers are available on your computer, it should appear as a standard serial interface. You should open this interface at a baud rate of `1000000bps`.
+Given that the appropriate drivers are available on your computer, it should appear as a standard serial interface. You should open this interface at a baud rate of `115200bps`.
```

Comment:
Changing the baud rate is going to cause an issue with identifying the firmware revision from the driver.

I agree with changing it to a more standard board rate though.

Is there another method we could use to determine the board firmware? Perhaps a USB descriptor? Worst case we could get another vid/pid issued.